src/search/components/SearchResult/index.js

```diff
@@ -26,16 +41,35 @@ export default class SearchResult extends React.Component {
   render() {
     const { result } = this.props;
     return (
-      <Link to={`/search/addons/${result.slug}`} className="search-result" ref="container">
-        <div className="search-result--name" ref="name">
-          {result.name}
+      <li className="SearchResult" ref="container">
+        <div>
+          <img
+            className="SearchResult-icon"
+            src={result.icon_url}
+            height="64"
+            width="64"
+            alt="Icon"
+          />
+        </div>
+        <div className="SearchResult-main">
```

Comment:
I could see the li being a sections instead but this doesn't encompass all the information about an add-on so that doesn't seem right to me.